server/test/handlers/checkAvailability.test.js

```diff
@@ -0,0 +1,97 @@
+const workerTimeslotHandler = require('../../models/handlers/workerTimeslot');
+
+describe('Test to check expected returns of checkAvailability functionality ', () => {
+    beforeEach(() => {
+        jest.resetModules();
+    });
+
+    ///TEST 1
+    test('rejection of null workerTimeslotId', async () => {
+        // Arrange
+        // No test variables need to be initialized.
+        const workerTimeslotId = null;
+
+        // Act
+        const workerIsAvailable = await workerTimeslotHandler.checkWorkerAvailability(workerTimeslotId);
+
+        // Assert
+        expect(workerIsAvailable).toBe(false);
+    });
+
+    //TEST 2
+    test('rejection of an empty string value for workerTimeslotId', async () => {
+        // Arrange
+        const workerTimeslotId = '';
+
+        // Act
+        const workerIsAvailable = await workerTimeslotHandler.checkWorkerAvailability(workerTimeslotId);
+
+        // Assert
+        expect(workerIsAvailable).toBe(false);
+    });
+
+    //TEST 3
+    test('rejection of an filled string value for workerTimeslotId', async () => {
+        // Arrange
+        const workerTimeslotId = 'good_day_sire';
+
+        // Act
+        const workerIsAvailable = await workerTimeslotHandler.checkWorkerAvailability(workerTimeslotId);
+
+        // Assert
+        expect(workerIsAvailable).toBe(false);
+    });
+    
+    //TEST 4
+    test('check valid workerTimeslotId for true return ', async () => {
+        // Arrange
+        const workerTimeslotId = 6;
+
+        // Act
+        const workerIsAvailable = await workerTimeslotHandler.checkWorkerAvailability(workerTimeslotId);
+
+        // Assert
+        expect(workerIsAvailable).toBe(true);
+    });
+
+    //TEST 5
+
+    /*PLEASE NOTE: Test case 5 is conditional provided that the status of the
```

Comment:
Hi Nancy,
Due to this limitation, I would suggest updating the database value so that others would not need to manually update their database in order to fix failing tests. To do this, I would recommend creating a migrations file named, e.g., `016.do.adjust-workerTimeslot-test-data` with the sql query to update that row, e.g., `UPDATE worker_timeslot set status='unavailable' where worker_timeslot_id=5;`.

Reply:
Hi Amy,
That's a great suggestion! It is actually exactly what I wanted to do before creating this PR :D However, upon consulting with Peter, I learned that he has created 2 migrations files called 0.16... and 0.17... 
To address this, I thought of 4 options, but they are all unfeasible:
a) If I were to create a file called 016.do.adjust-workerTimeslot-test-data, it would overrun his migration file.
b)I am not allowed to create a file out of order (eg. 018...)
c) I also suggested to add another entry to the existing migrations folder, however, I was recommenced to avoid making changes to the existing folders to prevent any breakage in master.
d) I requested that Peter add an entry in his migrations folder corresponding to an unavailable status, so that upon merging his file with master, my test code would be able to leverage that entry. However, his code actively changes the status from unavailable to available, so this would not make a difference.

As such, we decided that the best approach for now was to test the local instance where the workerTimeslotId corresponds to a status = unavailable, and ensure that the test code returned false. Thus, we know that the functionality is sound and works as expected.

Reply:
Hi Nancy, 
The current setup would permit the creation of two files starting with `0.16` as long as they do not have the same file name. I would suggest creating such a file to avoid having everyone manually adjust their database values in order to run their tests. The purpose of the addition to the migrations file is to ensure that this unit test does not break for other users. Although his code actively changes the values from `unavailable` to `available`, Jest resets the modules before running the next unit test. His adjustment would only be impactful if the change was made in the migrations file.
Hope this helps!

Reply:
Hi Amy,
Thank you for helping me navigate this problem! 
My Action: I have created a new migrations file with the name `016.do.adjust-workerTimeslot-test-data`

You can see this change in lines 1-2 of server/migrations/016.do.adjust-workerTimeslot-test-data
Please confirm if this takes care of your suggestion!

Reply:
Hi Nancy, 
As this is an sql file, I would recommend ending the file name with .sql so it properly runs.